account_global_discount/views/account_invoice_views.xml

```diff
@@ -0,0 +1,138 @@
+                    string="Global Discounts"
+                    invisible="not global_discount_ids_readonly or not invoice_global_discount_ids"
+                    groups="!base_global_discount.group_global_discount"
+                />
+            </field>
+            <field name="tax_totals" position="attributes">
+                <attribute
+                    name="class"
+                    add="oe_subtotal_footer_separator"
+                    separator=" "
+                />
+            </field>
+            <page id="other_tab" position="inside">
+                <separator string="Global Discounts" />
+                <field
+                    name="invoice_global_discount_ids"
+                    nolabel="1"
+                    readonly="1"
+                    invisible="not global_discount_ids"
+                    force_save="1"
+                    groups="base_global_discount.group_global_discount"
+                >
+                    <tree create="0" delete="0">
+                        <field name="name" />
+                        <field name="currency_id" invisible="1" />
+                        <field name="global_discount_id" invisible="1" />
+                        <field name="discount" invisible="1" />
+                        <field
+                            name="base"
+                            widget="monetary"
+                            options="{'currency_field': 'currency_id'}"
+                        />
+                        <field name="discount_display" />
+                        <field name="discount_amount" />
+                        <field
+                            name="base_discounted"
+                            widget="monetary"
+                            options="{'currency_field': 'currency_id'}"
+                        />
+                        <field name="account_id" />
+                        <field name="tax_ids" widget="many2many_tags" />
+                        <field name="company_id" invisible="1" />
+                        <field
+                            domain="[('company_id', '=', company_id)]"
+                            name="account_analytic_id"
+                            groups="analytic.group_analytic_accounting"
+                        />
+                    </tree>
+                </field>
+                <field
+                    name="invoice_global_discount_ids"
+                    nolabel="1"
+                    readonly="1"
+                    invisible="not global_discount_ids_readonly"
+                    force_save="1"
+                    groups="!base_global_discount.group_global_discount"
+                >
+                    <tree create="0" delete="0">
+                        <field name="name" />
+                        <field name="currency_id" invisible="1" />
+                        <field name="global_discount_id" invisible="1" />
+                        <field name="discount" invisible="1" />
```

Comment:
The same as above

Reply:
@dalonsod You're absolutely right. Thanks for the correction. I've already fixed it.😊